lib/mongodb/instrumentation/command_subscriber.rb

```diff
@@ -48,8 +51,8 @@ def failed(event)
         # tag the reported duration and any error message that came through
         span = @requests[event.request_id]
         span.set_tag("took.ms", event.duration * 1000)
-        span.set_tag("error", true)
-        span.log_kv(key: "message", value: event.message)
+        error = Error.new(event.message) 
```

Comment:
This one (compared to the http status code Error allocation) seems OK to me though.